.gitignore

```diff
@@ -13,3 +13,6 @@
 /examples/simple.min.js
 contribs/gmf/fonts/FontAwesome*
 contribs/gmf/fonts/fontawesome*
+
+.idea/
+tags
```

Comment:
Better to start with a `/` to be sure that's come from the root folder (same for fonts)...